scheduler/services/resource/ocs_resource_service.py

```diff
@@ -515,6 +532,86 @@ def none_to_str(value) -> str:
                     s = self._negative_filters[site].setdefault(d, set())
                     s.add(LgsFilter())
 
+    def _parse_eng_task_file(self, site: Site, to_file: str) -> None:
+        """
+        Parse Engineering task that block moments or the entire night.
+        Each twilight is calculated using lucupy.sky, some discrepancies might affect.
+        """
+        pattern = r'(\[.*\])'
+        # Ignore GS until the file is created.
+        if site is Site.GS:
+            return
+
+        with open(os.path.join(self._path, to_file), 'r') as file:
+            for line in file:
+                # Find pattern to keep bracket comments not splitted
+                match = re.search(pattern, line)
+                if match:
+                    comment = match.group(1)
+                    rest_of_line = re.sub(pattern, '', line)
+                    _date, start_time, end_time = rest_of_line.strip().split()
+                    #  Single date for key
+                    just_date = datetime.strptime(_date, '%Y-%m-%d')
```

Comment:
Just curious why you're starting off local variable names with `_` here, especially if you're using them?